openedx/features/student_account/helpers.py

```diff
@@ -163,3 +165,13 @@ def check_and_add_third_party_params(request, params):
             params['provider'] = running_pipeline.get('backend')
 
         params['access_token'] = running_pipeline['kwargs']['response']['access_token']
+
+
+def get_registration_countries():
+    countries_dict = {'all_countries': sorted(COUNTRIES.items(), key=operator.itemgetter(1)),
```

Comment:
countries_options, please avoid appending datatype

Reply:
Approach changed, this is not valid anymore.